integrationtest/spring/model/src/main/java/acme/org/hibernate/search/integrationtest/spring/repackaged/model/MyOtherEntity.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package acme.org.hibernate.search.integrationtest.spring.repackaged.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+
+@Entity
+@Indexed
+public class MyOtherEntity {
```

Comment:
Maybe also include a projection class somewhere? After all, that's the only point of classpath scanning.